src/mme-app/msgHandlers/s10MsgHandler.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2019, Infosys Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <msgHandlers/s10MsgHandler.h>
+
+#include <contextManager/subsDataGroupManager.h>
+#include <event.h>
+#include <eventMessage.h>
+#include <ipcTypes.h>
+#include <log.h>
+#include <mmeSmDefs.h>
+#include <utils/mmeCommonUtils.h>
+#include "mmeStatsPromClient.h"
+
+using namespace SM;
+using namespace mme;
+using namespace cmn;
+
+S10MsgHandler::~S10MsgHandler() {
+
+}
+
+S10MsgHandler::S10MsgHandler()
+{
+
+}
+
+S10MsgHandler* S10MsgHandler::Instance()
+{
+	static S10MsgHandler msgHandler;
+	return &msgHandler;
+}
+
+void S10MsgHandler::handleS10Message_v(IpcEMsgUnqPtr eMsg)
```

Comment:
Are you adding new container ? Currently we have s11-app and are you planning to add s10-app ?